baker/GrapherImageBaker.tsx

```diff
@@ -119,8 +123,14 @@ export function initGrapherForSvgExport(
     jsonConfig: GrapherInterface,
     queryStr: string = ""
 ) {
+    // Applying this transform before baking is only a temporary measure for the
+    // svg tester, and can be removed once the grapher configs used for the svg tester
+    // are migrated using MigrateSelectedData1661264304751.
+    const converted = MigrateSelectedData1661264304751.transformConfig(
+        jsonConfig as LegacyGrapherInterface
+    )
```

Comment:
As the comment rightly says, this code is only temporary to run the SVG tester on migrated configs. We can remove it once the configs are updated in the database and an update of https://github.com/owid/owid-grapher-svgs has been run.